client/state/products-list/selectors.js

```diff
@@ -33,3 +43,40 @@ export function getProductDisplayCost( state, productSlug ) {
 
 	return product.cost_display;
 }
+
+export const getPlanPrice = ( state, siteId, plan, isMonthlyPreference ) => {
+	const isMonthly = isMonthlyPreference && plan.term !== TERM_MONTHLY;
+	return (
+		getPlanDiscountedRawPrice( state, siteId, plan.getStoreSlug(), { isMonthly } ) ||
+		getPlanRawPrice( state, plan.getProductId(), isMonthly )
```

Comment:
We could also consider updating getPlanRawPrice to handle these additional cases (never divide on a monthly plan) to update the interface to suit our new needs.

`getPlanRawPrice` should also have access to plan details in global state in addition to any plan constants